src/components/sharedDiary/SharedDiaryFunnel.tsx

```
@@ -0,0 +1,93 @@
+      //     text={text}
+      //     file={file}
+      //     onSubmit={() => {
+      //       // TODO: 업로드 API 호출
+      //       funnel.history.push("complete");
+      //     }}
+      //     onBack={() => funnel.history.back()}
+      //   />
+      // );
+    }
+
+    case "complete": {
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)